packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -0,0 +1,198 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/breadcrumbs';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Breadcrumbs} from '@react-spectrum/breadcrumbs';
+import {Flex} from '@react-spectrum/layout';
+import {Item} from '@react-stately/collections';
+```
+
+# Breadcrumbs
+
+<p>{docs.exports.Breadcrumbs.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Breadcrumbs']} />
+
+## Example
+
+```tsx example
+<Breadcrumbs>
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+  <Item uniqueKey="Folder 3">Folder 3</Item>
+</Breadcrumbs>
+```
+
+## Content
+
+Breadcrumbs accept as list of `children`, each with a `uniqueKey` prop.
+
+It is highly suggested the `children` are Item components from the `@react-stately/collections` library.
+
+```tsx example
+function Example() {
+  return (
+    <Breadcrumbs>
+      <Item uniqueKey="Folder 1">Folder 1</Item>
+      <Item uniqueKey="Folder 2">Folder 2</Item>
+      <Item uniqueKey="Folder 3">Folder 3</Item>
+    </Breadcrumbs>
+  );
+}
+```
+
+In order to identify children when a user takes an action on a item,
+each child needs to have a `uniqueKey` prop. See [Events](#events) for more information.
+
+It is possible (but not suggested) to use alternative components as children,
```

Comment:
I'd remove this for now in case we end up using CollectionBuilder to implement at some point.

Reply:
only Items, since all children are passed to the menu component (using CollectionBuilder) for collapsed breadcrumbs variant...